src/Data/Dwarf/Internals.hs

```diff
@@ -9,6 +11,16 @@ module Data.Dwarf.Internals
   , whileJust
   , strictGet
   , getWhileNotEmpty
+  , Sections(SectionContents)
+  , requiredSection
+  , dsStrSection,
+  dsAbbrevSection,
+  dsInfoSection,
+  dsRangesSection,
+  dsLineSection,
+  dsLineStrSection,
+  dsStrOffsets,
+  dsAddr
```

Comment:
Style nit pick: be consistent with the existing conventions in this export list:

```suggestion
  , dsStrSection,
  , dsAbbrevSection
  , dsInfoSection
  , dsRangesSection
  , dsLineSection
  , dsLineStrSection
  , dsStrOffsets
  , dsAddr
```